qt_gui/src/qt_gui/plugin_handler_direct.py

```diff
@@ -122,7 +121,7 @@ def _restore_settings(self, plugin_settings, instance_settings):
                 self._plugin.restore_settings(plugin_settings_plugin, instance_settings_plugin)
             except Exception:
                 qCritical(
-                    'PluginHandlerDirect._restore_settings() plugin "%s" raised an exception:\n%s' %
+                    'PluginHandlerDirect._restore_settings() plugin "{}" raised an exception:\n' %
```

Comment:
Changing to ``{}`` requires that you call `format` on the string. eg: ``"example one: {} two: {}".format("1", "2")`
Also, this will omit the `traceback.format_exc()`` argument. 

Should be:
``'PluginHandlerDirect._restore_settings() plugin "{}" raised an exception:\n{}'.format(str(self._instance_id), traceback.format_exc())`` 
or
``'PluginHandlerDirect._restore_settings() plugin "%s" raised an exception:\n %s' % (str(self._instance_id), traceback.format_exc())``

Reply:
Whoops, yah this was a mistake